client/my-sites/themes/theme-showcase.jsx

```diff
@@ -213,6 +231,7 @@ class ThemeShowcase extends React.Component {
 						uri={ this.constructUrl() }
 					/>
 				) }
+				<SampleBanner />
```

Comment:
Up to you, but you could also simply inline

```js
<ThemesBanner
	title={ translate( 'Are you a small business?' ) }
	/* etc */
/>
```

here. (But I guess it's for sake of the example :slightly_smiling_face: )

Reply:
Not sure I understand what you mean here, sorry :)

And yes, it's for the sake of the example, but I'd like to know for future reference. Mind expanding on this one?

Reply:
Sure -- I was merely suggesting to drop `<SampleBanner />` here and directly use

```js
<ThemesBanner
	title={ translate( 'Are you a small business?' ) }
	/* etc */
/>
```

😄

(I think the extra variable doesn't necessarily add to legibility.)